src/components/Auth/Login/Login.js

```diff
@@ -0,0 +1,148 @@
+import { useState, useContext } from "react";
+
+import { Link, useNavigate } from "react-router-dom";
+
+import { UserContext } from "../../../store/user/userContext";
+import * as userActionTypes from "../../../store/types/userActionType";
+import { showToast } from "../../../utils/helper";
+import "../auth.css";
+
+export default function Login({ from }) {
+  const validators = {
+    email: (email) => {
+      const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+      return re.test(String(email).toLowerCase());
```

Comment:
Instead of re you can give this variable a meaningful name.

Reply:
Hi Kajol, 

Thanks for taking time to review, I have made the necessary changes. Please check it once.